packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -101,6 +102,15 @@ + (ComponentDescriptorProvider)componentDescriptorProvider
   return concreteComponentDescriptorProvider<ViewComponentDescriptor>();
 }
 
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    
+    if(_props->transformOrigin.length() > 0){
+        NSString *transformOrigin = [NSString stringWithCString: _props->transformOrigin.c_str() encoding:[NSString defaultCStringEncoding]];
+        applyTransformWithTransformOrigin(self, transformOrigin, self.layer.transform);
+    }
+}
```

Comment:
Why does this need to be overridden? RN does not use layoutSubviews, as layout happens async.

Reply:
We need to re-calculate the new transform-origin if the layout changes and re-apply transforms. Is there a better place to do the same? I set it here thinking it'll still trigger if RN sets the view's bounds, no? Also related - https://github.com/facebook/react-native/pull/37606#discussion_r1210073039 can you specify which hooks? 

Thanks for the quick review! I'll push the mentioned changes soon.

Reply:
For the new architecture, you'll want to do this from `updateLayoutMetrics` I believe

Reply:
Thanks, for old arch I switched to `reactSetFrame`.